### PR TITLE
Added example responses and updated styling to endpoint pages

### PR DESCRIPTION
**Pull request by rav-pradhan**
### What

- Updated the styling for the responses pages to reflect that which is on the prototype.
- Added `ExampleResponse` property to `MethodResponse` struct and `json` marshal code to render example responses onto the endpoint pages.

### How to review

- Pull changes and visit an endpoint page, e.g., http://localhost:23600/filter/filter-outputs-filter_output_id/
- Check that styling aligns with that of the prototype
- Check example response rendering as expected

### Who can review

Anyone but me.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -286,9 +288,13 @@ func generateMethods(path openAPI.PathItem) (methods []PathMethod) {
 
 func generateResponses(responses *openAPI.Responses) (orderedResponses []MethodResponse) {
 	for status, response := range responses.StatusCodeResponses {
+
+		json, _ := json.MarshalIndent(response.ResponseProps.Schema, "", "  ")
```
Comment (daiLlew) [inline main.go]: Do you need to check for the potentially error here? No sure what we would need to do if this does fail but worth considering.

Comment (rav-pradhan) [reply main.go]: Thanks; makes sense to check for an error in the marshal process. Have updated to include an `err` variable and if statement that calls `log.Fatal` to reflect the way errors are handled elsewhere in the file

File: main.go [hunk 2]
```diff
@@ -120,29 +121,29 @@ func main() {
 	}
 
 	if err := sources.Load(); err != nil {
-		log.Fatal(err)
+		log.Event(nil, "Failed to load sources", log.Error(err))
 	}
 
 	siteModel := generateModel(sources)
-	fmt.Println("Creating files...")
+	log.Event(nil, "Creating files...")
```
Comment (daiLlew) [inline main.go]: Nothing functionally wrong with this but in general I think its good practice to include as much relevant info in your log message as possible. If this blew on prod logs are your friend in working out whats going on - what files are being created here?

File: main.go [hunk 3]
```diff
@@ -120,29 +121,29 @@ func main() {
...
 		if err = renderer.Render(file, value.templateName, value); err != nil {
-			log.Fatal(err)
+			log.Event(nil, "Failed to render templates", log.Error(err))
 		}
 	}
 
-	fmt.Println("Files created")
-	fmt.Println("Finished!")
+	log.Event(nil, "Files created.")
+	log.Event(nil, "Finished!")
```
Comment (daiLlew) [inline main.go]: Remove the second log as its not really adding anything.

File: main.go [hunk 4]
```diff
@@ -286,9 +287,18 @@ func generateMethods(path openAPI.PathItem) (methods []PathMethod) {
 
 func generateResponses(responses *openAPI.Responses) (orderedResponses []MethodResponse) {
 	for status, response := range responses.StatusCodeResponses {
+
+		json, err := json.MarshalIndent(response.ResponseProps.Schema, "", "  ")
+
+		if err != nil {
+			log.Event(nil, "Failed to marshall json", log.Error(err))
```
Comment (daiLlew) [inline main.go]: What failed to marshall to json?

File: main.go (hunk 2 above)
Comment (rav-pradhan) [reply main.go]: Thanks for this - have updated these as per discussion

**Pull request merged**